core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -43,13 +47,32 @@ contract Liquidatable is PricelessPositionManager {
         FixedPoint.Unsigned liquidatedCollateral;
         // Unit value (starts at 1) that is used to track the fees per unit of collateral over the course of the liquidation.
         FixedPoint.Unsigned rawUnitCollateral;
-        /** Set upon initiation of a dispute */
+        /** Following variable set upon initiation of a dispute */
```

Comment:
For consistency I think you should use double slashes instead of slash star: `//` vs `/** **/`

Reply:
ye, I just want to differentiate that they are "sections" added a slightly different format using an arrow:
```
struct LiquidationData {
        // -> Following variables set upon creation of liquidation
        address sponsor; // Address of the liquidated position's sponsor
        address liquidator; // Address who created this liquidation
        Status state; // Liquidated (and expired or not), Pending a Dispute, or Dispute has 
		...
	}
```